okhttp/src/test/java/okhttp3/internal/tls/CertificatePinnerChainValidationTest.java

```diff
@@ -120,7 +128,7 @@
   /** The pinner should accept an intermediate from the server's chain. */
   @Test public void pinIntermediatePresentInChain() throws Exception {
     // TODO https://github.com/square/okhttp/issues/4703
-    assumeFalse(getJvmSpecVersion().equals("11"));
+    //jdkMatchRule.expectFailure(fromMajor(11), anything());
```

Comment:
why commented out code?

Reply:
Yeah, TLS issue should be resolved after OpenJDK 11.0.3. I guess we can safely remove the TODO comment and `expectFailure` related to 4703? :)

Reply:
Happy to remove.  I might leave the comment without "TODO" and remove the dead code.  Essentially someone running with 11.0.1 will see test failures and I don't want those reported.